### DISCUSSION TITLE
How to set up to make these symbols display normally?

### DISCUSSION THREAD

**Discussion by NyoronGero** [sumatrapdfreader/sumatrapdf#3596] [Q&A]
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/48859558/a8247978-13b7-4648-ac42-e5ad2f5c6a78)
↑ the effect of reading the epub file online at [app.flowoss.com](https://app.flowoss.com/) (display correctly)

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/48859558/829fc5f0-886b-4056-a60f-22ba0dc9e549)
↑ the effect of reading the epub file online at [neat-reader.cn](neat-reader.cn) (display correctly)

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/48859558/311b0183-c9ff-46dd-9cd6-9c8ab5a46178)
↑ the effect of reading the epub file in SumatraPDF

All adopt the default settings, without modifying the typesetting and font.
The encoding of the question symbol is **\u0025**(UTF-16).

Another problem:
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/48859558/a41cc451-c17e-4489-88a9-65945acc6dd2)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/48859558/3389de37-a43c-47db-9f88-c970a7d9905f)
Symbols used for vertical typesetting should not be converted to horizontal typesetting symbols. 
I hope that the text typesetting direction can be selected in the view options.

**Comment by dungsaga** [comment]
This looks like a problem with library MuPDF in SumatraPDF. You may have the same problem with other apps that also use library MuPDF.

**Comment by kjk** [comment]
Can't investigate without the file showing the problem.

**Reply by NyoronGero** [reply]
> The encoding of the question symbol is \u0025(UTF-16).
